<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_popup_PurchaseList_Pre</class>
 <widget class="QMainWindow" name="frm_popup_PurchaseList_Pre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>采购项</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="le_popup_PurchaseList_Name">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_popup_PurchaseList_Pre_Name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>模块名称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_popup_PurchaseList_Status">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_popup_PurchaseList_Status">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_popup_PurchaseList_Commiter">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>提交人</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_popup_PurchaseList_Approver">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>审核人</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_popup_PurchaseList_Commiter">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_popup_PurchaseList_Approver">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_popup_PurchaseList_Pre_Save">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>280</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_popup_PurchaseList_Pre_Cancel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>280</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
